csfieldguide/curriculum_guides/content/en/ncea/ncea.md

```diff
@@ -5,31 +5,8 @@ for the following NCEA Achievement Standards.
 
 {panel type="caution" expanded="true"}
 
-# These guides may be out of date.
+# These guides have moved!
 
-We recommend reading the guidance on this page very carefully to ensure students are working on the right material.
-There is also a lot of material available online from NZQA, and good ideas shared on the [DTTA](http://nzacditt.org.nz/) mailing list.
+They will return on the [DTHM for kaiako site](https://www.dthm4kaiako.ac.nz/).
```

Comment:
When 3.0 is out I imagine the guides will already be on DTHM site? 

If so, could we change the wording to "They can now be found on ... ".
Would also be good to update the link once they are out. However I'm not entirely sure when they will be put on the site.

Reply:
> For the CG PR, I'm happy to merge it in as is and add the dthm4kaiako.ac.nz links just before release.